doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -92,6 +92,8 @@ Rewriting with Leibniz and setoid equality
 
      If :n:`at @occs_nums` is specified, rewriting is always done with
      :ref:`setoid rewriting <generalizedrewriting>`, even for Leibniz’s equality.
+     However, note that `rewrite ... at` and `setoid_rewrite` don't behave identically.
+     See `#13728 <https://github.com/coq/coq/issues/13728>`_.
```

Comment:
From the fact that you link to this issue and that you do not mark the PR as closing it, I deduct that you think that something else would need to be done to close it. But this is not how I see things. To me this is just a documentation issue and therefore this PR is enough to fix it.

Reply:
OK.   BTW "deduct" means "subtract or take away (an amount or part) from a total".  I think you want "deduce".

Reply:
Yep. Kind of the same mistake as "imply" and "implicate" :laughing:

Reply:
Interestingly and perhaps confusingly, "deduction" is related to both "deduct" and "deduce":

1. the action of deducting or subtracting something
2. the inference of particular instances by reference to a general law or principle.